src/dodal/devices/synchrotron.py

```diff
@@ -1,9 +1,23 @@
 from enum import Enum
 
 from ophyd import Component, Device, EpicsSignal
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_r
 
 
-class SynchrotronMode(Enum):
+class PV(str, Enum):
+    STATUS_PREFIX = "CS-CS-MSTAT-01:"
+    TOP_UP_PREFIX = "SR-CS-FILL-01:"
+    SIGNAL_PREFIX = "SR-DI-DCCT-01:"
+    SIGNAL = "SIGNAL"
+    MODE = "MODE"
+    USRCNTDN = "USERCOUNTDN"
+    BEAM_ENERGY = "BEAMENERGY"
+    CNTDN = "COUNTDOWN"
+    ENDCNTDN = "ENDCOUNTDN"
```

Comment:
Any particular reason? - I did it at the time to make the tes structure more iterative, but that has since become less of an issue

Reply:
Nit: My 2 cents is I feel like pulling them into constants in general is overkill. We haven't in any other device, do we think it should be the standard? (I particularly dislike `CNTDN = "COUNTDOWN"` where it feels like we've named the constant to be less clear than just the string). But take or leave it, minor nit pick really

Reply:
> I guess it depends how long the non-OASynchrotron will persist. If we're all agreed that it can be done away with, then each value is only being used once and don't pollute the namespace.

Yh, I'll make a Hyperion PR to use this instead. We also have some generic "wait for topup" plans that I'll move into dodal.

Reply:
Decided to go for the two enums approach to simplify importing in the test and prevent name clashes with the test data, as discussed.